cvat/apps/organizations/views.py

```diff
@@ -55,7 +55,6 @@ class OrganizationViewSet(viewsets.ModelViewSet):
     ordering_fields = filter_fields
     ordering = '-id'
     http_method_names = ['get', 'post', 'patch', 'delete', 'head', 'options']
-    pagination_class = None
```

Comment:
@k1won , It isn't enough. UI doesn't expect the pagination list of organizations. Need to change UI as well. Please let me know if you can do that.

Reply:
@k1won , I will ask to handle the issue inside my team. I hope it will be fixed in our next release.
I will close the PR for now.

Reply:
@nmanovic, ok.